File: codekata/absolute beginner/leap-year.py
A=int(input())
# To check Leap Year or not
if A % 4 == 0 and A % 100 != 0:
    print('Y')
elif A % 100 == 0:
    print('N')
elif A % 400 ==0:
    print('Y')
else:
    print('N')

File: javascript/codekata/absolute beginner/leap_year.js
const readline=require('readline');
const inp=readline.createInterface({
  input:process.stdin});
const n=[];
inp.on("line",(data)=>{
  n.push(data);});
inp.on("close",()=>{
  var year=parseInt(n[0]);
  if((year%4)==0){
    if((year%100)==0){
      if((year%400)==0){
        console.log("Y")}
      else{
        console.log("N")}}
    else{
      console.log("Y")}}
  else{
    console.log("N")}

});
